core-blocks/paragraph/index.js

```diff
@@ -251,7 +197,7 @@ class ParagraphBlock extends Component {
 							fallbackBackgroundColor,
 							fallbackTextColor,
 						} }
-						isLargeText={ fontSize >= 18 }
+						isLargeText={ fontSize.size >= 18 }
```

Comment:
Should this be built-in to the abstraction if "large" text (for accessibility compliancy) is a standard condition?

Reply:
Yes maybe we can add a property of fontSize to contrast checker where we only pass the fontSize and it decides if it is large or not, but as it is not directly related to changes here I will do that in a different PR.

Reply:
> but as it is not directly related to changes here I will do that in a different PR.

Can you create a (ideally self-assigned) issue tracking this?

Reply:
For this cases where I feel a PR would be very small I prefer to directly create a PR instead of writting an issue. The PR is availabel at https://github.com/WordPress/gutenberg/pull/8059/files.

It endup being a little bigger than what I was expecting initally because I needed to add additional test cases.